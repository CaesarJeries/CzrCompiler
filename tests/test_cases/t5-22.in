void main() {

    int x = 0 + 200b * 2b + 1;
    printi(x);
    print("\n");

    printi(126 / (250b + 2b * 3b));
    print("\n");

}