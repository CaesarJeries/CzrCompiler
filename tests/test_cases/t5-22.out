Loaded: ./exceptions.s
145
Error division by zero
